R/data_tabulate.R

```diff
@@ -16,7 +16,10 @@
 #' @param name Optional character string, which includes the name that is used
 #' for printing.
 #' @param remove_na Logical, if `FALSE`, missing values are included in the
-#' frequency or crosstable, else missing values are omitted.
+#' frequency or crosstable, else missing values are omitted. Note that the
+#' default for the `as.table()` method is `remove_na = TRUE`, so that missing
+#' values are not included in the returned table, which makes more sense for
+#' post-processing of the table, e.g. using `chisq.test()`.
```

Comment:
Sounds weird to have different default values, but maybe the main usecase for calling `as.table()` is to run a test right after. Would it make sense to add a message "Dropping NAs" by default in `as.table()`?

Reply:
Ok, then I'd also add a `verbose` argument